cpp/mrc/src/tests/test_network.cpp

```diff
@@ -84,12 +85,21 @@
 using namespace mrc;
 using namespace mrc::memory::literals;
 
+class DataPlaneResources2Tester : public data_plane::DataPlaneResources2
+{
+  public:
+    std::shared_ptr<runtime::RemoteDescriptorImpl2> get_descriptor(uint64_t object_id)
+    {
+        return m_remote_descriptor_by_id[object_id];
+    }
```

Comment:
We should discuss ways to avoid this pattern. Feel free to change visibility of any method as you see fit.

Reply:
What would be the preferred solution here? Do you prefer that we make `m_remote_descriptor_by_id` public or something?

Reply:
As we discussed, this is likely a fine solution.